src/Entity/OuterExtension/HasEmail.php

```diff
@@ -22,9 +22,9 @@ public function getEmail()
     /**
      * Set email
      *
-     * @param object $email
+     * @param Email
```

Comment:
Ah OK. Precisely, it has seemed me strange to do not indicate it. Pushed done again.